crates/sui-indexer-alt/src/db.rs

```diff
@@ -38,7 +39,7 @@ pub struct DbConfig {
     /// Time spent waiting for a connection from the pool to become available.
     #[arg(
         long,
-        default_value = stringify!(DEFAULT_CONNECTION_TIMEOUT_SECS),
+        default_value = const_format!("{DEFAULT_CONNECTION_TIMEOUT_SECS}"),
```

Comment:
Drop the dep on const_str and instead use the "default_value_t" parameter for setting a typed default vs a stringly-typed one

Reply:
We can't do this because the type is a `Duration` which does not implement `clap`'s `AsValue`.

Reply:
I would probably just parse a u64 then and convert to Duration at the place its needed vs bringing in another dependency to do this but i'll leave that up to you.

Reply:
Yes, I think I'll end up doing this as part of a larger refactor to enable using file-based configs (which I think are unfortunately inevitable when we start thinking about pruning configurations etc).

Reply:
thanks for the fix! will the file-based config have a default u64 value, in that sense we will still have 2 sources of the default values as before, one in the config file and the other in this file like below?
```
const DEFAULT_CONNECTION_TIMEOUT_SECS: u64 = 60;
```

Reply:
The aim is to avoid having multiple sources of truth.